apps/server/src/services/__tests__/timerUtils.test.ts

```diff
@@ -21,17 +21,17 @@ describe('getExpectedFinish()', () => {
       timer: {
         addedTime: 0,
         duration: 10,
-        finishedAt: null,
         startedAt: null,
       },
       _timer: {
         pausedAt: null,
+        hasFinished: false,
```

Comment:
question: is there a reason to keep `finishedAt` as a number? it would allow us to make calculations that imply knowing the offset since finish time. Nothing springs to mind to me....

Reply:
not as fare as i could see either 
and then we can just put it back as it in internal to the server now